cmd/tsbs_generate_queries/databases/clickhouse/devops.go

```diff
@@ -0,0 +1,341 @@
+package clickhouse
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/devops"
+	"github.com/timescale/tsbs/query"
+)
+
+// Devops produces ClickHouse-specific queries for all the devops query types.
+type Devops struct {
+	*devops.Core
+	UseTags bool
+}
+
+// NewDevops makes an Devops object ready to generate Queries.
+func NewDevops(start, end time.Time, scale int) *Devops {
+	return &Devops{
+		devops.NewCore(start, end, scale),
+		false,
+	}
+}
+
+// GenerateEmptyQuery returns an empty query.ClickHouse
+func (d *Devops) GenerateEmptyQuery() query.Query {
+	return query.NewClickHouse()
+}
+
+// getHostWhereWithHostnames creates WHERE SQL statement for multiple hostnames.
+// NB 'WHERE' itself is not included, just hostname filter clauses, ready to concatenate to 'WHERE' string
```

Comment:
`NB` -> `NOTE:`

Reply:
Ok, updated, just used to write **NB** as shortcut for **nota bene** from students' time. May be it is shorter